samples/local_attestation/common/attestation.h

```diff
@@ -17,28 +17,25 @@ class Attestation
   public:
     Attestation(Crypto* crypto, uint8_t* enclave_mrsigner);
 
-    // Generate a report for the given data. The SHA256 digest of the
-    // data is stored in the report_data field of the generated report.
-    bool generate_local_report(
+    // Generate evidence for the given data.
+    bool generate_local_attestation_evidence(
         uint8_t* target_info_buffer,
         size_t target_info_size,
         const uint8_t* data,
         size_t data_size,
-        uint8_t** report_buf,
-        size_t* remote_report_buf_size);
+        uint8_t** evidence_buf,
```

Comment:
OE style: avoid abbreviations, so as long as you're touching this anyway, change "buf" to "buffer" like line 22 already uses